tests/keras/utils/data_utils_test.py

```diff
@@ -33,17 +32,15 @@
                                      'TRAVIS_PYTHON_VERSION' in os.environ,
                                      reason='Generators do not work with `spawn`.')
 
-if sys.version_info < (3,):
-    def next(x):
-        return x.next()
-
 
 def use_spawn(func):
-    """Decorator to test both Unix (fork) and Windows (spawn)"""
+    """Decorator which uses `spawn` when possible.
+    This is useful on Travis to avoid memory issues.
+    """
 
     @six.wraps(func)
     def wrapper(*args, **kwargs):
-        if sys.version_info > (3, 4):
+        if sys.version_info > (3, 4) and os.name != 'nt':
```

Comment:
So it's not possible to use spawn if on windows with python 3.4+? I thought that spawn was the only way to create a new process on windows. Maybe I'm missing something.

Reply:
`spawn` is the default on windows. So if we call `mp.set_start_method('fork', force=True)` on Windows, it would fail because it doesn't support the `fork` pattern.

Reply:
Right, my bad.